### PR TITLE
Update Arduino IDE

### PR DESCRIPTION
**Pull request by JL2210**

### CODE REVIEW

File: packages/arduino_ide.rb [hunk 1]
```diff
@@ -19,15 +22,18 @@ class Arduino_ide < Package
...
-      platform = 'linux32'
+      @platform = 'linux32'
     when 'armv7l', 'aarch64'
-      platform = 'linuxarm'
+      @platform = 'linuxarm'
     end
+
+  def self.build
+    version = @version
+    platform = @platform
```
Comment (uberhacker) [inline packages/arduino_ide.rb]: These variable assignments shouldn't be necessary.  The instance variable defined at the top could be used anywhere in the class; including functions and methods.

File: packages/arduino_ide.rb [hunk 2]
```diff
@@ -40,26 +46,21 @@ def self.build
...
-      platform = 'linux32'
-    when 'armv7l', 'aarch64'
-      platform = 'linuxarm'
-    end
+    version = @version
+    platform = @platform
     Dir.chdir("build") do
       FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"
-      system "tar", "xpvf", "linux/arduino-#{version}-#{platform}.tar.xz", "-C", "#{CREW_DEST_PREFIX}/share/"
+      system "tar", "-xpf", "linux/arduino-#{version}-#{platform}.tar.xz", "-C", "#{CREW_DEST_PREFIX}/share/"
```
Comment (uberhacker) [inline packages/arduino_ide.rb]: The dash is not necessary on the tar command.

Comment (uberhacker) [reply packages/arduino_ide.rb]: And why did you remove the verbose option?

File: packages/arduino_ide.rb (hunk 1 above)
Comment (JL2210) [reply packages/arduino_ide.rb]: Done.

File: packages/arduino_ide.rb (hunk 2 above)
Comment (JL2210) [reply packages/arduino_ide.rb]: Done. Did not add back `-v` option as not needed.

**Comment by JL2210** [comment]
@uberhacker This is ready.

File: packages/arduino_ide.rb [hunk 3]
```diff
@@ -19,47 +22,41 @@ class Arduino_ide < Package
...
     when 'x86_64'
-      platform = 'linux64'
+      @platform = 'linux64'
     when 'i686'
-      platform = 'linux32'
+      @platform = 'linux32'
     when 'armv7l', 'aarch64'
-      platform = 'linuxarm'
+      @platform = 'linuxarm'
     end
```
Comment (uberhacker) [inline packages/arduino_ide.rb]: This section should be moved up above the description since they are instance variables.  Please use proper indentation.

**Comment by JL2210** [comment]
@uberhacker This is ready.

**Pull request merged**